config/ModulesMapping.jsonc

```diff
@@ -1,4 +1,5 @@
 {
+  "Admin": "^admin",
```

Comment:
`admin` tag is already being used by modules such as `Devices.ServiceAnnouncement` and `WindowsUpdates`. To avoid duplicate commands, only add `admin.*` children to the appropriate module. For example, `admin.adminReportSettings` tag should be under `Reports` module.

Reply:
I have added it under Reports